.circleci/config.yml

```diff
@@ -10,70 +11,73 @@ jobs:
       tag: "17.0"
     steps:
       - checkout
-      - run: mkdir -p ~/build-artifacts
+      - run: mkdir -p ./build-artifacts
       - gradle/with_cache:
           steps:
             - run: ./gradlew build -x test
       - run:
-          name: Copy JARs to build-artifacts
-          command: |
-            shopt -s globstar
-            cp **/build/libs/*.jar ~/build-artifacts
+          name: Show the JARs
+          command: ls -l build-artifacts
       - persist_to_workspace:
-          root: ~/
+          root: ./
           paths:
             - build-artifacts
 
   smoke_test:
-    docker:
-      - image: cimg/base:stable
-    environment:
-      AGENT_JAR_GLOB: $HOME/build-artifacts/honeycomb-opentelemetry-javaagent*-all.jar
-      AGENT_ONLY_APP_JAR_GLOB: $HOME/build-artifacts/spring-agent-only-?.?.?.jar
-      AGENT_MANUAL_APP_JAR_GLOB: $HOME/build-artifacts/spring-agent-manual-?.?.?.jar
+    machine:
+      image: ubuntu-2004:202107-02
     steps:
-      - attach_workspace:
-          at: ~/
-      - setup_remote_docker:
-          version: 20.10.7
       - checkout
+      - attach_workspace:
+          at: ./
+      - bats/install
```

Comment:
Use the orb to get the BATS.

![bats](https://user-images.githubusercontent.com/517302/142068201-3e472bcb-fdb8-4942-8d25-24b8cab6a38e.gif)